Restoring Backup with an Existing Seed.md

```diff
@@ -0,0 +1,78 @@
+# Restoring Backup with an Existing Seed
+
+If you have already owned a Skywallet to store your coins and have bought a new one to replace it, you can opt to restore the old Skywallet using the new one.
+
+In order to restore your coins, you would only need the Seed of the old Skywallet.
+
+***A Seed is a combination of 12 or 24 words which serves as the key to access your Skywallet.***
+
+**NOTE - You can only use the Seed of a Skywallet on another Skywallet and cannot use the Seed of a Desktop Wallet on a Skywallet.**
```

Comment:
Please delete. It was my mistake, you actually can use the desktop wallet  seed.